content/en/network_monitoring/cloud_network_monitoring/guide/detecting_application_availability.md

```diff
@@ -0,0 +1,125 @@
+
+## Visibility into network traffic 
+
+Datadog CNM consolidates relevant distributed traces, logs, and infrastructure data into a single view, allowing you to identify and trace issues back to the originating request from an application.
+
+In the example below, check the Traces tab under Network Analytics to view distributed traces of requests between source and destination endpoints, which can help you pinpoint where application-level errors occur.
+
+To identify if an issue is an Application or Network issue, you can use the following steps:
+
+1. Navigate to [Infrastructure > Cloud Network > Analytics][1].  
+2. In the Summary graphs, click a line of communication that has a lot of volume and high RTT time:
+
+   {{< img src="network_performance_monitoring/guide/detecting_network_insights/cnm_isolate_series.png" alt="CNM analytics page, clicking on a path with high RTT Time">}}
+
+3. Click **Isolate this series**  
+4. Click into this path and click on the Flows tab to observe RTT time:
+
+   {{< img src="network_performance_monitoring/guide/detecting_network_insights/cnm_sidepanel_rtt.png" alt="CNM sidepanel, highlighting the RTT time column">}}
+
+5. On this page, CNM correlates network metric round-trip time (RTT) with application request latency, to help us identify if the issue is a network or application issue. In this particular example, observe that the RTT time is slightly high but has come down over time and needs to be investigated further.  
+     
+6. On this same page, click the Traces tab and investigate the Duration column: 
+
+   {{< img src="network_performance_monitoring/guide/detecting_network_insights/cnm_traces_duration.png" alt="CNM sidepanel, highlighting the Traces tab and duration column">}}
+
+   Observe that although network latency (RTT) is high, the application request latency (Duration) is normal, so in this case, the issue is likely network-related, and there's no need to investigate the app code. 
+
+   Conversely, *if network latency is stable but application latency (Duration) is high*, the problem likely stems from the app, and you can explore code-level traces by clicking on one of the service paths in the Traces tab to find the root cause, which takes you to the APM flame graph relative to this service:
+
+   {{< img src="network_performance_monitoring/guide/detecting_network_insights/cnm_apm_traces.png" alt="APM flame graph screenshot after clicking on a service from the CNM sidepanel traces tab">}}
+
+### Network map
+
+The [Network map][2] in Datadog provides a visual representation of your network topology, helping identify partitions, dependencies, and bottlenecks. It consolidates network data into a directional map, making it easier to isolate problematic areas. Additionally, it visualizes network traffic between any tagged object in your environment, from `services` to `pods` to cloud regions.
```

Comment:
Should `cloud regions` be formatted in monospace too? Do "services" and "pods" need to be monospaced at all? Later on, it looks like at least "pods" lose the formatting, so maybe it can be safely removed here too.

Reply:
when we referring to a tag such as a `service` tag or `pod` tag it does need monospace, so yes cloud regions should have that too thanks for the catch!